ext/standard/tests/network/bug68925.phpt

```diff
@@ -2,12 +2,18 @@
+try {
+    var_dump(gethostbyname(str_repeat("0", 2501)));
+} catch (\ValueError $e) {
+    echo $e->getMessage() . \PHP_EOL;
+}
+try {
+    var_dump(gethostbynamel(str_repeat("0", 2501)));
+} catch (\ValueError $e) {
+    echo $e->getMessage() . \PHP_EOL;
+}
 
-Warning: gethostbynamel(): Host name is too long, the limit is %d characters in %s%ebug68925.php on line %d
-bool(false)
+?>
+--EXPECT--
+gethostbyname(): Argument #1 ($hostname) cannot be longer than 255 characters
+gethostbynamel(): Argument #1 ($hostname) cannot be longer than 255 characters
```

Comment:
These should be `%d`. Though I'm not really sure if we actually want to promote these. The number differs by platform and is something one could hit with reasonable code. As `gethostbyname()` is already fallible, it would currently be a necessary false check.